src/Plugin/Util/Plugin.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace PHPCensor\Plugin\Util;
+
+class Plugin
```

Comment:
I think class name should be more clear. Maybe `BitbucketPluginResult` or `PluginResult`.